lib/licensee/license.rb

```diff
@@ -63,7 +63,7 @@ def licenses
     # `no-license` - The project is not licensed (e.g., all rights reserved)
     #
     # Note: A lack of detected license will be a nil license
-    PSEUDO_LICENSES = %w[other no-license].freeze
+    PSEUDO_LICENSES = %w(other no-license).freeze
```

Comment:
You're probably using a recent but not current version of rubocop. This was fixed/changed back in https://github.com/bbatsov/rubocop/pull/4231